### ISSUE TITLE
relevance_features standardization

### ISSUE THREAD

**Issue by OuyKai** [su-zhan/Graph4DIV#1]
Is relevance_features standardized? If standardization operation is adopted, what standardization method is used

**Comment by su-zhan** [comment]
We use the same relevace features as DSSA, for we do not use the relevance features of subtpopics, we only use part of the files. The normalized function is as follows.
'''
import pandas as pd
from sklearn.preprocessing import StandardScaler
rel_feat = pd.read_csv('data/rel_feat.csv')
rel_feat_names = list(sorted(set(rel_feat.columns) - {'query', 'doc'}))
rel_feat[rel_feat_names] = StandardScaler().fit_transform(rel_feat[rel_feat_names])
rel_feat = dict(zip(map(lambda x: tuple(x), rel_feat[['query', 'doc']].values),
                    rel_feat[rel_feat_names].values.tolist()))
'''

> Is relevance_features standardized? If standardization operation is adopted, what standardization method is used

**Comment by OuyKai** [comment]
Thanks
